pkg/asset/agent/image/oc.go

```diff
@@ -188,8 +188,13 @@ func (r *release) extractFileFromImage(image, file, cacheDir string) (string, er
 	if err != nil {
 		return "", err
 	}
-	// set path
 	path := filepath.Join(cacheDir, path.Base(file))
+	// Make sure file exists
```

Comment:
Could the file have existed before running the extraction? We may need to check and delete it first.

Reply:
IINW the check if there was already the iso in the cache is performed before reaching it, so it should be ok

Reply:
GetFileFromCache() https://github.com/openshift/installer/blob/master/pkg/asset/agent/image/oc.go#L111 does already check for the file.

Reply:
But if `verifyCacheFile()` returns false then we will proceed to this code despite the existence of an old file.
And in fact it will always return false when we have the wrong release payload, because it calls this function to get the SHA sum, which will return an error: https://github.com/openshift/installer/blob/master/pkg/asset/agent/image/oc.go#L260-L264

Reply:
That's true, we could get a false positive with new existence check.
Add a change to remove the file in case it exists before extraction.
Tested with both failing and passing cases.